src/test/kotlin/step3_racing_car/game/ui/InputConsoleTest.kt

```diff
@@ -0,0 +1,39 @@
+package step3_racing_car.game.ui
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.NullAndEmptySource
+import org.junit.jupiter.params.provider.ValueSource
+
+class MockReceiver : InputReceiver {
+    var answer: String? = "0"
+    override fun receive(): String? {
+        return answer
+    }
+}
+
+internal class InputConsoleTest {
+    val mockReceiver = MockReceiver()
+
+    @Test
+    fun runTest() {
+        // given
+        mockReceiver.answer = "1"
+        assertThat(InputConsole(mockReceiver).run()).isEqualTo(Pair(1, 1))
+    }
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    @ValueSource(strings = ["0", "-1", "#", "$"])
+    fun exceptionTest(input: String?) {
```

Comment:
```suggestion
    fun exceptionTest(input: String) {
```

nullable하게 선언안해도 괜찮아보여요!

Reply:
음 이부분은 null 테스트를 일부러 한 부분이라 nullable 이어야 할 것 같습니다.